diskimage_builder/elements/centos8/post-install.d/23-cloud-init-default-user

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if [ ${DIB_DEBUG_TRACE:-0} -gt 0 ]; then
+    set -x
+fi
+set -eu
+set -o pipefail
+
+# Change the name of the default user if requested.
+if [ -n "${DIB_CLOUD_INIT_DEFAULT_USER:-}" ]; then
+        # Changes the default cloud-init user
+        sed -i -e 's/cloud-user/'$DIB_CLOUD_INIT_DEFAULT_USER'/g' /etc/cloud/cloud.cfg
```

Comment:
Here the sed match is very broad. 

You should match `name: cloud-user` and replace by `name: $DIB_CLOUD_INIT_DEFAULT_USER` e.g.:

`sed -i -e "s/name: cloud-user/name: $DIB_CLOUD_INIT_DEFAULT_USER/g" /etc/cloud/cloud.cfg`